### DISCUSSION TITLE
Followers/Following tab issue

### DISCUSSION THREAD

**Discussion by sakshi-keepworks** [community/community#37446] [Other Feature Feedback, Questions, & Ideas]
On [this](https://github.com/sakshi-keepworks?tab=following) page, when I click on some following user's name, it does not redirect the whole page to that user's profile. The left sidebar still remains on current user's page. The same issue is with followers tab also.

https://user-images.githubusercontent.com/4617609/198205972-450db68f-20fb-4bb7-9440-f0b4981630bb.mp4

**Comment by ossanna16** [comment]
Hi there @sakshi-keepworks and welcome to our community! Thank you for reporting this 🙂

**Comment by radar-blip** [comment]
Is fixed now.

**Comment by jfullstackdev** [comment]
yup, it's fixed by now, actually several days ago
